examples/4/Verifier.toml

```diff
@@ -0,0 +1 @@
+setpub = []
```

Comment:
Do we want to add all these files or remove them?

Reply:
Yep, I left these in so that people can also prove and verify with the examples